lib/KettleConfigLoader.js

```diff
@@ -101,7 +101,7 @@ kettle.config.createDefaultsImpl = function (prefix, filePath) {
         fullPath = path.resolve(prefix, fileName);
     }
     var configFile;
-    var parser = kettle.JSON.readFileSync(fullPath, "reading config file at " + fullPath);
+    var parser = kettle.readFileSync(fullPath, "reading config file at " + fullPath);
```

Comment:
Please add a test using a JSON5-formatted config file (converting one of the existing ones to JSON5 is fine).
Please also add relevant explanation to docs/ConfigsAndApplictions.md - that is, at relevant times where it says "JSON" write "JSON or JSON5" together with a link to http://json5.org/